### ISSUE TITLE
Feature Request - Add Labels for Displays

### ISSUE THREAD

**Issue by varon** [koekeishiya/yabai#1616]
It would be very useful from a scripting perspective to be able to assign a label to a display and use that in DISPLAY_SEL.

For instance, 'left' or 'right' for displays is a lot easier to reason about than magical UUIDs, especially for commands involving sending spaces and windows to specific displays.

**Comment by brorbw** [comment]
I use
```sh
yabai -m display --focus west  
yabai -m display --focus south 
yabai -m display --focus north 
yabai -m display --focus east  
```
And equally for moving windows
```sh
yabai -m window --display west 
yabai -m window --display south
yabai -m window --display north
yabai -m window --display east 
```
If you already have it in a script. I guess you could just assign it to something else
```sh
LEFT=<UUID-FOR-LEFT-SCREEN>
RIGHT=<UUID-FOR-RIGHT-SCREEN>
```

**Comment by varon** [comment]
Thank you, @brorbw. That's a good set of suggestions.

Moving windows between displays is covered using the tools you've suggested, but sometimes I have need to move across multiple displays to get to the target.

I'm using screens by UUID currently for their various references, but this means I still have to store mappings between the UUID and 'human readable' display names - We're already attaching labels to spaces, so why not just add them as an optional field/reference in yabai?

**Comment by brorbw** [comment]
This should cover moving across multiple displays
```
alt - h: yabai -m window --focus west  || (yabai -m display --focus west  && yabai -m window --focus east)
alt - j: yabai -m window --focus south || (yabai -m display --focus south && yabai -m window --focus north)
alt - k: yabai -m window --focus north || (yabai -m display --focus north && yabai -m window --focus south)
alt - l: yabai -m window --focus east  || (yabai -m display --focus east  && yabai -m window --focus west)
```
Equally for moving windows:
```
shift + cmd - h: yabai -m window --swap west  || (yabai -m window --display west && yabai -m display --focus west)
shift + cmd - j: yabai -m window --swap south || (yabai -m window --display south && yabai -m display --focus south)
shift + cmd - k: yabai -m window --swap north || (yabai -m window --display north  && yabai -m display --focus north)
shift + cmd - l: yabai -m window --swap east  || (yabai -m window --display east && yabai -m display --focus east)
```

**Comment by koekeishiya** [comment]
On master it is possible to change how arrangement indices are interpreted (e.g. sort by x-coordinate/y-coordinate): https://github.com/koekeishiya/yabai/issues/550#issuecomment-1989676801

This should allow for a more intuitive usage of default indices (1 to display_count).
I am not against adding labels if that is still regarded as a useful addition.

**Comment by koekeishiya** [comment]
Implemented labels for displays on master.

**Comment by koekeishiya** [comment]
Fixed in v7.0.0.
